### PR TITLE
Majoron script update

### PR DESCRIPTION
**Pull request by ashleyrback**
Modifies the `Xe136_majoron_limit` script in light of some changes introduced
in PR #76.

Continuing, for now, with the assumption that KamLAND-Zen has the same expected
solar rate as SNO+, updates the `B8_Solar` prior to be the expected rate for the
full 0-15 MeV range from SNO+-doc-507. Note this number has been updated in
SNO+-doc-507v27 for the whitepaper.

### CODE REVIEW

File: echidna/scripts/Xe136_majoron_limit.py [hunk 1]
```diff
@@ -51,17 +51,52 @@ def main(args):
...
+    phase_space = 4.54e-17
+    matrix_element = 1.e-3
+    nuclear_params.append((phase_space, matrix_element))
+
+    livetime = 112.3 / 365.25  # y, KamLAND-Zen 112.3 live days
+
     # Apply FV and livetime cuts
-    fv_radius = 1200.0  # 1.2m PRC 86, 021601 (2012)
+    fv_radius = 1200.  # 1.2m PRC 86, 021601 (2012)
     livetime = 1.0
```
Comment (jwaterfield) [inline echidna/scripts/Xe136_majoron_limit.py]: Two values of livetime. Here and L87. One needs to be removed

File: echidna/scripts/Xe136_majoron_limit.py [hunk 2]
```diff
@@ -51,17 +51,52 @@ def main(args):
...
+    matrix_element = None
+    nuclear_params.append((phase_space, matrix_element))
+    # n=3:
+    phase_space = 1.06e-17  # Assuming two Majorons emitted
+    matrix_element = 1.e-3
+    nuclear_params.append((phase_space, matrix_element))
+    # n=7:
+    phase_space = 4.54e-17
+    matrix_element = 1.e-3
+    nuclear_params.append((phase_space, matrix_element))
```
Comment (jwaterfield) [inline echidna/scripts/Xe136_majoron_limit.py]: Is there a reference for the above numbers (L71 to here)?

**Comment by jwaterfield** [comment]
The following lines need changing to meet pep8 standard

L94:80: E501 line too long (81 > 79 characters)
L98:80: E501 line too long (81 > 79 characters)
L190:5: E303 too many blank lines (2)

File: echidna/scripts/Xe136_majoron_limit.py (hunk 2 above)
Comment (ashleyrback) [inline echidna/scripts/Xe136_majoron_limit.py]: Yes, I'll add them.

File: echidna/scripts/Xe136_majoron_limit.py (hunk 1 above)
Comment (ashleyrback) [inline echidna/scripts/Xe136_majoron_limit.py]: Thanks, I'll fix that.

**Comment by ashleyrback** [comment]
@jwaterfield, that should fix all the comments, let me know if there is anything else.

**Comment by jwaterfield** [comment]
Happy merging

**Pull request merged**